.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,12 @@
-# What / Why
-<!-- Describe the request in detail -->
-> n/a
+<!-- What / Why >
```

Comment:
Is the `>` meant to be a closing comment tag? (ie. `-->`)

Reply:
Oh, yes! Thanks 🙂